end-to-end-test/end_to_end_test/test_queue_worker.py

```diff
@@ -28,9 +29,9 @@ def test_queue_worker(cog_server, project_dir, redis_port, tmpdir_factory):
     input_queue = multiprocessing.Queue()
     output_queue = multiprocessing.Queue()
     controller_port = find_free_port()
-    local_ip = get_local_ip()
-    upload_url = f"http://{local_ip}:{controller_port}/upload"
-    redis_host = local_ip
+    bridge_ip = get_bridge_ip()
```

Comment:
I'm not 100% sure this change will be stable on other machines, so would appreciate if someone else could checkout this branch and run the e2e tests to check!